metafacture-xml/src/main/java/org/metafacture/xml/SimpleXmlEncoder.java

```diff
@@ -192,11 +211,11 @@ public void endEntity() {
 
     @Override
     public void literal(final String name, final String value) {
-        if (name.isEmpty()) {
+        if (name.isEmpty() || name.equals(valueTag)) {
```

Comment:
We could also set `DEFAULT_VALUE_TAG = ""` and drop `name.isEmpty()`.

Reply:
+1 simplify if you can